src/components/LanguageSettings.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from "react";
```

Comment:
there's too many language settings in this app.
can we rename it to something more specific?